bpf/mock/Makefile

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017-2020 Authors of Cilium
```

Comment:
Probably should use 2021 here and in other license headers :)

Reply:
Thanks! I changed the license for bpf/mock/Makefile. For mock/helpers.h, mock/helpers_skb.h and mock/helpers_xdp.h, they are generated from the original files in include/bpf/, so their licenses are consistent.